#this should contain ALL information about the runtime
bool gh_present
bool agent_present
int64 ros_tick
string current_controller
my_msgs/state state
my_msgs/action action